example/.coveralls.yml

```diff
@@ -1,2 +1,2 @@
 # Uncomment the following line for private GitHub repositories.
-# service_name: travis-pro
+# service_name: ci-pro
```

Comment:
This "service name" refers to a Coveralls configuration (https://docs.travis-ci.com/user/coveralls/), and there is no such thing as `ci-pro`. Please see if our current Coveralls integration can be maintained without Travis CI. If not, we'll need to discuss that. (Also, be careful not to change code until you understand what it means in the first place, and confirm that your change makes sense in context. 😉)